test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -14,26 +15,47 @@ import {
 
 jest.mock("src/realtimePrice", () => require("test/mocks/realtimePrice"))
 
-const defaultWallet = ""
-
 describe("Bitcoind", () => {
   it("create default wallet", async () => {
     try {
-      const { name } = await bitcoindClient.createWallet(defaultWallet)
+      const { name } = await bitcoindClient.createWallet("default")
       // depends of bitcoind version. needed in < 0.20 but failed in 0.21?
-      expect(name).toBe(defaultWallet)
+      expect(name).toBe("default")
+    } catch (error) {
+      baseLogger.warn({ error }, "bitcoind wallet already exists")
+    }
+    const wallets = await bitcoindClient.listWallets()
+    expect(wallets).toContain("default")
+  })
+
+  // TODO: only to be considered in the rest of the tests
+  it("create hot wallet", async () => {
```

Comment:
this should be refactored, this is the same code as the `default` wallet above

Reply:
and outside also use the same code

Reply:
Refactor done with createWalletInClient function. And also added an initial test that checks that no wallet is created yet to remove the original try-catch for the error "bitcoind wallet already exists".

Reply:
this `it` also isn't a test (no `expect`) so needs to be part of setup